src/Entities/Player.cpp

```diff
@@ -2655,12 +2655,12 @@ float cPlayer::GetLiquidHeightPercent(NIBBLETYPE a_Meta)
 
 bool cPlayer::IsInsideOfBlock(BLOCKTYPE a_Block)
 {
-	BLOCKTYPE Block = m_World->GetBlock(GetPosX(), m_Stance, GetPosZ());
+	BLOCKTYPE Block = m_World->GetBlock((int)GetPosX(), (int)m_Stance, (int)GetPosZ());
```

Comment:
`(int)` is used in c `and static_cast<int>()` is the replacement for it in c++. Also the compiler would warn of use of `old-style cast`, if there is no flag to ignore it.